datafusion/proto/tests/cases/roundtrip_logical_plan.rs

```diff
@@ -704,6 +730,7 @@ async fn roundtrip_logical_plan_distinct_on() -> Result<()> {
     let plan = ctx.sql(query).await?.into_optimized_plan()?;
 
     let bytes = logical_plan_to_bytes(&plan)?;
+    println!("plan {:?}", plan);
```

Comment:
nit: could you remove this debug println?

Reply:
oops